### DISCUSSION TITLE
Add Imba support & docs

### DISCUSSION THREAD

**Discussion by unmeimusu** [withastro/docs#3169] [Ideas]
Imba is like pyscript and provides full-stack features. 
If you want to check it here is the link below.
https://imba.io/

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
